package.json

```diff
@@ -49,6 +49,7 @@
     "babel-eslint": "10.0.2",
     "babel-plugin-filter-imports": "3.0.0",
     "babel-plugin-istanbul": "5.1.4",
+    "babel-plugin-minify-replace": "^0.5.0",
```

Comment:
Prefer exact versions. You can do this by using `--exact` with your `yarn add` command.